### DISCUSSION TITLE
1/3

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#37] [Announcements]
# 1/3

SICP Python 中文翻译

https://sicp.csfive.works/1/3

**Comment by FDULeolu** [comment]
1.3.1里面 global frame的翻译全局帧是标准术语翻译嘛？如果不是的话我从小白的角度感觉翻译成全局框架更合适一点？更接近一个环境的概念

**Reply by kobayashilin1** [reply]
应该翻译为：全局帧，在Python中，每次函数调用都会创建一个帧，该帧包含了函数的局部变量、参数、返回地址等信息。
